lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -476,6 +476,33 @@ private JourneyResponse buildF2FNoMatchResponse(
         return JOURNEY_F2F_FAIL;
     }
 
+    private JourneyResponse buildDCMAWContinuationResponse(
+            VerifiableCredentialBundle credentialBundle,
+            Vot lowestGpg45ConfidenceRequested,
+            ClientOAuthSessionItem clientOAuthSessionItem,
+            AuditEventUser auditEventUser)
+            throws IpvSessionNotFoundException, VerifiableCredentialException {
+        var criResponseItem =
+                criResponseService.getCriResponseItem(
+                        clientOAuthSessionItem.getUserId(), DCMAW_ASYNC);
+        var previousIpvSessionItem =
+                ipvSessionService.getIpvSessionByCriOAuthSessionId(criResponseItem.getOauthState());
```

Comment:
Is there a chance this might be missing/expired in which case we need some safety around it?

Reply:
I think you are right. I have added a null check for the CriResponseItem